dts/bindings/radio_fem/skyworks,sky66112-11.yaml

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) 2020 Nordic Semiconductor ASA
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+
+description: >
+    This is a representation of the Skyworks SKY66112-11 Radio Front-End module.
+
+compatible: "skyworks,sky66112-11"
+
+include: base.yaml
+
+properties:
+    ctx-gpios:
+        type: phandle-array
```

Comment:
Thank you for making these phandle-arrays here. That's better than `ctx-pin` values. We'd like to get away from `foo-pin` properties in general, even though we use them sometimes in Nordic IP blocks because it makes it more convenient to initialize PSEL registers. The phandle array is better because it refers to the actual GPIO controller by phandle, which the tooling can see.  Since the tools know a GPIO controller, this will enable future changes that let us build a pin map for the SoC in a generic way. The `foo-pin` type options don't let the tooling know that the property is actually configuring a pin.